doc/01_overview/targets.rst

```diff
@@ -7,8 +7,6 @@ ASIC Synthesis
 ASIC synthesis is supported for Ibex.
 The whole design is completely synchronous and uses positive-edge triggered flip-flops, except for the register file, which can be implemented either with latches or with flip-flops.
 See :ref:`register-file` for more details.
-The core occupies an area of roughly 24 kGE when using the latch-based register file and implementing the RV32IMC ISA, or 15 kGE when implementing the RV32EC ISA.
-
```

Comment:
Removing this sounds good to me, especially since we report area numbers in README.md already. I do think we should have some numbers somewhere but I agree that we don't want to keep updating numbers in several places.